Marlin/src/inc/SanityCheck.h

```diff
@@ -4563,6 +4569,15 @@ static_assert(_PLUS_TEST(4), "HOMING_FEEDRATE_MM_M values must be positive.");
   #endif
 #endif
 
+/**
+ * Fixed-time-based Motion Control requirements
+ */
+#if ENABLED(FT_MOTION)
+  #if ENABLED(I2S_STEPPER_STREAM)
+    #error "FT_MOTION is not supported with I2S_STEPPER_STREAM."
```

Comment:
The `FT_MOTION` option may not completely work with it, but I see there actually is a block for the `I2S_STEPPER_STREAM` within `Stepper::shaping_isr`, so maybe that ought to be removed.

Of course, both Input Shaping and Fixed-Time Motion can be disabled at runtime, and it might be possible to run non-shaping steppers in some relation to I2S. The documentation within Marlin on the `I2S_STEPPER_STREAM` is not great, so it would help to have a better understanding of what it does, and whether it can also be switched on/off at runtime to accommodate internal shaping as needed.